.prettierignore

```diff
@@ -2,3 +2,4 @@ dist
 node_modules
 build
 .docusaurus
+packages/docusaurus-utils/lib/
```

Comment:
Why do we Prettier ignore this? Will all TS files be Prettier ignored?

Reply:
lib is the compiled file